libexec/scrape

```diff
@@ -17,6 +19,8 @@ done
 
 IFS=: read -ra DEF_PATHS <<<"${RUBY_BUILD_DEFINITIONS:-share/ruby-build}"
 
+RUBIES=( ${RUBIES[@]:- rubinius} )
```

Comment:
This feels syntactically clumsy. How about simply:

[ ${#RUBIES[@]} -gt 0 ] || RUBIES=(rubinius)